### PR TITLE
cli v4 migration guide

### PR DESCRIPTION
**Pull request by jault3**

### CODE REVIEW

File: guides/cli-migration-v4.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+category: guide
+---
+
+# Compliant Cloud CLI Version 4
+
+The release of version 4.0.0 of the Compliant Cloud CLI brings major changes to existing commands as well as changes in CLI workflows. This guide will walk through upgrading to version 4 and explain the changes that it brings.
+
+## Upgrading to version 4
+
+Upgrading to CLI version 4 does not happen automatically or by running the `datica update` command. Please visit the [releases page](https://github.com/daticahealth/cli/releases) to download the latest version of the CLI. Unzip the package and replace your old CLI binary with the new one. You can find the location of your existing CLI binary by running `which datica`. After upgrading, running `datica --version` should reflect the version downloaded.
```
Comment (nathanjsweet) [inline guides/cli-migration-v4.md]: Maybe you should add a note that automatic downloads will work after the one time manual update? What do you think?

File: guides/cli-migration-v4.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+1. Run `datica init`
+1. Run `git push datica master`
+
+The `init` command performs all the same functionality as the `associate` command and more. Specifically, the `init` command will create a git repository in the current directory if one does not exist, add the proper `datica` git remote for the chosen code service, and add an SSH key to your user account if one was not found.
+
+The `init` command should only be run for setting up new environments or for setting up a local development environment. For all other purposes you can run the CLI commands directly by specifying `-E` with the full environment name.
+
+## 32-bit support removed
+
+Datica no longer publishes 32-bit binaries. If you are in need of a 32-bit version of the CLI please [contact Datica support](https://datica.com/support).
```
Comment (nathanjsweet) [inline guides/cli-migration-v4.md]: Or you can build the cli yourself? That might not be worth adding though as people who know how to do that can think of it for themselves.

Comment (jault3) [inline guides/cli-migration-v4.md]: Yeah I would be fine just letting those users build it on their own.

File: guides/cli-migration-v4.md (hunk 1 above)
Comment (jault3) [inline guides/cli-migration-v4.md]: I will add that

**Comment by nathanjsweet** [comment]
LGTM

**Pull request merged**